app/views/email_alerts/medical_safety_alerts/publication.html.erb

```diff
@@ -15,6 +15,12 @@
         <%= document_summary %>
       </p>
 
+      <% if updated_or_published == "updated" && document_change_note.present? %>
```

Comment:
Hmm, I think I was just being over defensive. I don't believe there is a situation where there would be a document_change_note and we didn't want to see it.